src/components/Picker/index.js

```diff
@@ -102,10 +104,28 @@ class Picker extends PureComponent {
         };
 
         this.onInputChange = this.onInputChange.bind(this);
+        this.placeholder = this.props.placeholder;
+        this.items = this.props.items;
     }
 
     componentDidMount() {
         this.setDefaultValue();
+
+        // Windows will reuse the text color of the select for each one of the options
+        // so we might need to color accordingly so it doesn't blend with the background.
+        if (getOperatingSystem() === CONST.OS.WINDOWS) {
+            this.placeholder = {
+                ...this.placeholder,
+                color: '#002140',
+            };
```

Comment:
We need to first check if `this.props.placeholder` is and empty object, otherwise this will create an empty placeholder and in certain places it will lead to a crash (see video below)
```suggestion
            this.placeholder = _.isEmpty(this.placeholder) ? {} : {
                ...this.placeholder,
                color: '#002140',
            };
```

https://user-images.githubusercontent.com/22219519/204682628-308b1697-e9ec-4178-92c8-635ad937fbe0.mov